packages/cli/index.js

```diff
@@ -61,16 +65,61 @@ export async function run(...args) {
   }
 
   try {
-    await $`${binaryPath} ${args}`;
+    await spawnAsync(binaryPath, args, { stdio: 'inherit' });
```

Comment:
Not sure if this will actually work correctly, it doesn't seem like `spawn` takes a callback directly, so I think it's not really being promisified. From the few tests I did locally, it seemed like the Promise was never resolving